pysaliency/external_datasets/nusef.py

```diff
@@ -117,20 +122,28 @@ def get_NUSEF_public(location=None):
                          eye_scn_dist,
                          no_of_flags,
                          fix_loss,
-                         interfix_loss) = line.split()
+                         interfix_loss) = lines[i].split()
                         x.append(float(hor_pos) * scale_x)
                         y.append(float(ver_pos) * scale_y)
-                        t.append(float(start_time.split(':')[-1]))
+                        if i == 0:
+                            initial_minute = float(start_time.split(':')[-2])
+                        current_minute = float(start_time.split(':')[-2])
```

Comment:
Why do you parse the stuff manually? It should be much more concise to use `datetime.time`, either via  `strptime`  or in the worst case manually by passing hours, minutes and seconds. Also, right now the code has a bug if the scanpath starts, e.g. at `14:59:58`, because also the minutes will reset to zero.